frontend/packages/ceph-storage-plugin/src/utils/bucket-class.ts

```diff
@@ -14,3 +14,8 @@ export const validateDuration = (ms: number): boolean => ms >= 900000 && ms <= 8
 
 export const convertToMS = ({ unit, value }) =>
   unit === TimeUnits.HOUR ? parseInt(value, 10) * 3600000 : parseInt(value, 10) * 60000;
+
+export const convertMsToUnit = (unit: string, value: number): number =>
```

Comment:
```suggestion
export const convertToMs = (unit: TimeUnits, value: number): number =>
```

Reply:
@bipuladh It's not converting to ms, instead it's converting back ms to required unit(hr or min).

Reply:
My bad lets rename it to `convertTime`

Reply:
sure